integration_tests/upgrade/test-upgrade-1.2-to-1.3.sh

```diff
@@ -0,0 +1,1660 @@
+check_response "$result" "\"subject\": \"$test_root_cert_subject_new\""
+check_response "$result" "$test_root_cert_subject_key_id_new"
+
+echo "Get proposed x509 root certificate"
+result=$($DCLD_BIN_NEW query pki proposed-x509-root-cert --subject=$google_root_cert_subject_new --subject-key-id=$google_root_cert_subject_key_id_new)
+check_response "$result" "\"subject\": \"$google_root_cert_subject_new\""
+check_response "$result" "\"subjectKeyId\": \"$google_root_cert_subject_key_id_new\""
+check_response "$result" "\"vid\": $google_root_cert_path_new_random_vid"
+
+echo "Get revoked x509 certificate"
+result=$($DCLD_BIN_NEW query pki revoked-x509-cert --subject=$intermediate_cert_subject_new --subject-key-id=$intermediate_cert_subject_key_id_new)
+check_response "$result" "\"subject\": \"$intermediate_cert_subject_new\""
+check_response "$result" "\"subjectKeyId\": \"$intermediate_cert_subject_key_id_new\""
+
+echo "Get proposed x509 root certificate to revoke"
+result=$($DCLD_BIN_NEW query pki proposed-x509-root-cert-to-revoke --subject=$test_root_cert_subject_new --subject-key-id=$test_root_cert_subject_key_id_new)
+check_response "$result" "\"subject\": \"$test_root_cert_subject_new\""
+check_response "$result" "\"subjectKeyId\": \"$test_root_cert_subject_key_id_new\""
+
+echo "Get revocation point"
+result=$($DCLD_BIN_NEW query pki revocation-point --vid=$vid_new --label=$product_label_new --issuer-subject-key-id=$issuer_subject_key_id)
+check_response "$result" "\"vid\": $vid_new"
+check_response "$result" "\"issuerSubjectKeyID\": \"$issuer_subject_key_id\""
+check_response "$result" "\"label\": \"$product_label_new\""
+check_response "$result" "\"dataURL\": \"$test_data_url\""
+
+echo "Get revocation points by issuer subject key id"
+result=$($DCLD_BIN_NEW query pki revocation-points --issuer-subject-key-id=$issuer_subject_key_id)
+check_response "$result" "\"vid\": $vid_new"
+check_response "$result" "\"issuerSubjectKeyID\": \"$issuer_subject_key_id\""
+check_response "$result" "\"label\": \"$product_label_new\""
+check_response "$result" "\"dataURL\": \"$test_data_url\""
+
+echo "Get all proposed x509 root certificates"
+result=$($DCLD_BIN_NEW query pki all-proposed-x509-root-certs)
+check_response "$result" "\"subject\": \"$google_root_cert_subject_new\""
+check_response "$result" "\"subjectKeyId\": \"$google_root_cert_subject_key_id_new\""
+
+echo "Get all revoked x509 root certificates"
+result=$($DCLD_BIN_NEW query pki all-revoked-x509-root-certs)
+check_response "$result" "\"subject\": \"$root_cert_subject_new\""
+check_response "$result" "\"subjectKeyId\": \"$root_cert_subject_key_id_new\""
+
+echo "Get all proposed x509 root certificates to revoke"
+result=$($DCLD_BIN_NEW query pki all-proposed-x509-root-certs-to-revoke)
+check_response "$result" "\"subject\": \"$test_root_cert_subject_new\""
+check_response "$result" "\"subjectKeyId\": \"$test_root_cert_subject_key_id_new\""
+
```

Comment:
Please make sure that the new call (`x509-cert --subject-key-id=...`) works after upgrade

Reply:
Please look at`integration_tests/upgrade/test-upgrade-1.2-to-1.3.sh:835`